Scripts/run-linter.sh

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 echo -e "👉 Running SwiftFormat Linting"
 
 echo -e "👉 Linting Sources..."
-mint run swiftformat --lint --config .swiftformat Sources --exclude **/Generated,Sources/StreamChat/StreamStarscream,Sources/StreamChatUI/StreamNuke,Sources/StreamChatUI/StreamSwiftyGif
+mint run swiftformat --lint --config .swiftformat Sources --exclude **/Generated,Sources/StreamChat/StreamStarscream,Sources/StreamChat/StreamULID,Sources/StreamChatUI/StreamNuke,Sources/StreamChatUI/StreamSwiftyGif
```

Comment:
nit: would be nice to have all 3rd parties in a separate folder so we don't need to update the linter next time (whenever it will be)

Reply:
Yeah, that was exactly my thought too. To move the 3rd parties under a separate folder so it would be even visible from the project structure, this source comes from the vendor.

On the other hand, that reshuffling would over-complicate this PR == that work can follow as a standalone PR to avoid PR scope creep.

Reply:
Moving them to another folder means that we would not be having a folder per target, which is something desirable. This something I want to bring up as well wrt the tests being in the Sources folder. It would simplify things, like generation of excluded files in the Package.swift